app/controllers/application_controller.rb

```diff
@@ -17,4 +16,13 @@ def current_admin?
     current_user && current_user.admin?
   end
 
+  def admin?
+    role == 2
+  end
+
+  def merchant?
+    role == 1
+  end
```

Comment:
this is so I could check and see if the current user was either an admin or a merchant. The controller would then direct them accordingly.

Reply:
for this the enums actually has a method called admin? and merchant? which will check based on the role provided